packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -2206,6 +2239,19 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
     }
   }
 
+  @override
+  void startLiveTextInput(SelectionChangedCause cause) {
+    if (!liveTextInputEnabled || widget.readOnly || widget.obscureText) {
```

Comment:
Are the checks on widget.readOnly and widget.obscureText redundant?  I think they're included in liveTextInputEnabled.